sdk/batch/azure-batch/azure/batch/operations/__init__.py

```diff
@@ -0,0 +1,21 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from ._operations import BatchOpsOperations
```

Comment:
Is the newly added `operations` folder used? Because there are already a `_operations` folder under `azure/batch`, this seems depulicated.

Reply:
Removed duplicated `operations` folders